src/prisma/schema.prisma

```diff
@@ -3268,13 +3268,13 @@ model ScoutWallet {
   createdAt      DateTime           @default(now())
   address        String             @unique
   scoutId        String             @db.Uuid
+  primary        Boolean            @default(false)
```

Comment:
we should add an index on this field, we'll probably want to use it in the queries to get your wallet

Reply:
Added  `@@index([primary, scoutId])`